pipeline/pipelines-ui/overlays/application/kustomization.yaml

```diff
@@ -0,0 +1,13 @@
+apiVersion: kustomize.config.k8s.io/v1beta1
+kind: Kustomization
```

Comment:
why only add Application CR for pipeline-ui instead of all components of pipeline?